controller/volume_controller_test.go

```diff
@@ -1460,7 +1461,10 @@ func (s *TestSuite) runTestCases(c *C, testCases map[string]*VolumeTestCase) {
 			n, err := lhClient.LonghornV1beta1().Nodes(TestNamespace).Create(context.TODO(), node, metav1.CreateOptions{})
 			c.Assert(err, IsNil)
 			c.Assert(n, NotNil)
-			nIndexer.Add(n)
+			err = nIndexer.Add(n)
+			if err != nil {
+				log.Println(err)
+			}
```

Comment:
It's better to use `c.Assert(err, IsNil)` instead so that the unit test won't continue. Similar below

Reply:
I mean all errors in the unit test (`controller/xxxxx_controller.test.go`). In this PR, they are all like:
```
err = nIndexer.Add(n)
if err != nil {
	log.Println(err)
}
```

Instead, it's better to modify it to this:
```
err = nIndexer.Add(n)
c.Assert(err, IsNil)
```

Reply:
ok got it